config.go

```diff
@@ -21,13 +21,11 @@
 package config
 
 import (
-	"fmt"
+	"io/ioutil"
 	"os"
-	"path"
-	"path/filepath"
-	"sync"
+	"strings"
 
-	flag "github.com/ogier/pflag"
```

Comment:
Why are we using this random fork of Steve Francia's `pflag` library? It's years behind upstream.

Steve's now a core member of the Go team at Google, and he's an upstanding package maintainer. If possible, we should use `spf13/pflag` directly.